spiel/st/m0t1fs_spiel_dix_common_inc.sh

```diff
@@ -121,8 +124,8 @@ spiel_cleanup()
 
 spiel_dix_repair_start()
 {
-echo $M0_SRC_DIR/utils/spiel/m0spiel $SPIEL_OPTS
-    $M0_SRC_DIR/utils/spiel/m0spiel $SPIEL_OPTS <<EOF
+echo "$M0_SRC_DIR"/utils/spiel/m0spiel "$SPIEL_OPTS"
```

Comment:
Don't include $SPIEL_OPTS in double quotes. You can ignore this Codacy error

Reply:
incorporated